tests/cpp/ml_demo/src/eeg_metrics.cpp

```diff
@@ -98,7 +98,7 @@ int main (int argc, char *argv[])
             if (!metric)
                 metric_type = "Relaxation";
             for (int classifier = (int)BrainFlowClassifiers::REGRESSION;
-                 classifier <= (int)BrainFlowClassifiers::SVM; classifier++)
```

Comment:
seems like I missed it in previous PR, I want to keep eeg_metrics files simple and wo loops and add another file for testing which has classifier and metric as cmd args

Reply:
this one should run all required tests in CI https://github.com/brainflow-dev/brainflow/blob/master/tests/cpp/ml_demo/src/eeg_metrics_ci.cpp eeg_metrics.cpp should be simple and run LR for Concentration and any other classifier for relaxation and it should be the same in all bindings